mercury.spec

```diff
@@ -1,7 +1,7 @@
-%global dl_version 2.0.0rc3
+%global dl_version 2.0.0
```

Comment:
I think we can get rid of this now and use `%{version}` in it's place.

Reply:
done